### PR TITLE
Adding localdownload option

### PR DESCRIPTION
**Pull request by ctolon22**
Adding an option for doing the download of the artifacts (windows exporter, nssm and prometheus) from where this role is run instead of the target VM (for private network customers)

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -45,3 +46,5 @@ prom_temp_dir: "%TEMP%"
 prom_app_dir: C:\Program Files\prometheus
 prom_data_dir: "%ProgramData%\\prometheus"
 prom_config_file: "{{ prom_data_dir }}\\prometheus.yml"
+
+localdownload: "false"
```
Comment (patrickeasters) [inline defaults/main.yml]: To be consistent with other vars, can we make this a boolean?
```
localdownload: false
```

File: tasks/prometheus.yml [hunk 2]
```diff
@@ -11,18 +11,42 @@
 - name: Install prometheus
   when: prom_installed_version.failed or prom_installed_version.stdout is not search("^prometheus, version " ~ prom_server_version)
   block:
+    - name: Download Prometheus locally
+      delegate_to: localhost
+      get_url: 
+        url: "{{ prom_server_package }}"
+        dest: "/tmp/prometheus.zip"
+      when: localdownload == "true"
```
Comment (patrickeasters) [inline tasks/prometheus.yml]: If we make this a bool, instances of this would become:
```
when: localdownload
```

File: tasks/prometheus.yml [hunk 3]
```diff
@@ -11,18 +11,42 @@
...
+        src: "/tmp/prometheus.zip"
+        dest: "{{ prom_temp_dir }}\\nssm.zip"
+      when: localdownload == "true"
     - name: Download Prometheus
       win_get_url:
         url: "{{ prom_server_package }}"
         checksum: "{{ prom_server_package_checksum }}"
         checksum_algorithm: sha256
         dest: "{{ prom_temp_dir }}\\prometheus.zip"
+      when: localdownload == "false"
```
Comment (patrickeasters) [inline tasks/prometheus.yml]: and this can become
```
when: not localdownload
```

**Review by patrickeasters — COMMENTED** [review]
Just a minor request, but this is great. Thanks!

File: defaults/main.yml (hunk 1 above)
Comment (ctolon22) [reply defaults/main.yml]: I have corrected all localdownload and, per [this link](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html#conditionals-based-on-variables), added with `| bool`
Let me know how you see it

Comment (patrickeasters) [reply defaults/main.yml]: The filter is fine to include. It'll catch a few more cases.

**Review by patrickeasters — APPROVED** [review]

**Pull request merged**